README.md

```diff
@@ -90,112 +95,246 @@ You can use one of the example frontends available at `./examples` directory.
-   - In the bottom of the toolbar on the left, click on "Personal Tools".
+Now an editor can :-
+- login to hydra and see the frontend in an iframe as if it was volto
+- browse in hydra your frontend page will change. 
+- browse in your frontend, hydra will change context.
+- add a page in hydra and it will appear.
+- edit a page and after you save it will reload the iframe and the changes will appear on your frontend.
+   - they will be able to add blocks the frontend specifies that it can support. 
+- remove a page.
+- all other volto features outside editing work the same.
 
-2. **Go to Preferences**: 
-   - From the Personal Tools menu, select "Preferences".
+### Level 2: Enable Frontend block selection and Quanta Toolbar
 
-3. **Change Frontend URL**: 
-   - In the Preferences section, you will find an option to select the Frontend URL.
-   - You can either select a frontend URL from the available options or type in a custom URL:
-     - To select a URL from the options, simply choose from the dropdown menu.
-     - To enter a custom URL, click on the toggle to make the input field appear and type in your desired URL.
+Add the `data-block-uid={<<BLOCK_UID>>}` attribute to your outer most container of the rendered block html.
 
-This allows you to switch seamlessly between different frontend URLs for testing or editing purposes.
+For example, let's say you are rendering a Teaser block as
 
-**Note**: Make sure the frontend URL is correct and accessible to avoid any CORS issues.
+``` html
+<div class="teaser">
+<img src="/big_news.jpg"/>
+<h2>Big News</h2>
+<div>Check out <b>hydra</b>, it will change everything</div>
+<div><a href="/big_news">Read more</a><div>
+</div>
+```
 
-### Level 1: Show changes after save
+you would change this to
 
-This is the most basic form of integration.
-You can 
-- add and remove pages
-- navigate either using your frontend or the AdminUI
-- edit a page, but only via the sidebar and you will only see changes after you click save
+``` html
+<div class="teaser" data-block-uid="....">
+<img src="/big_news.jpg"/>
+<h2>Big News</h2>
+<div>Check out <b>hydra</b>, it will change everything</div>
+<div><a href="/big_news">Read more</a><div>
+</div>
+```
 
-To do this you will include the hydra iframe bridge which creates a two way link between the hydra editor and your frontend.
+Now an editor can :-
+- click directly on your block to edit the block contents in the sidebar. 
+   - The block will appear highlighted and a quantatool bar will appear above it.
+- selecting a block in the sidebar will highlight that block on the frontend
+- naviate to the parent block ([TODO](https://github.com/collective/volto-hydra/issues/66)) 
+
+If are using a 3rd party library to render a block, then you might not be able to easily modify the markup 
+to put in the ```data-block-uid```. Instead you can use an alternative 
+comment syntax ([TODO](https://github.com/collective/volto-hydra/issues/113)) where a comment goes
```

Comment:
yeah after reading the readme and I think this is indeed needed if someone is using a library to render components and have no direct access to html nodes. This issue might be little simpler than it looks hopefully lets try to include it in scope of this PoC.